### PR TITLE
New subsection describing enhancement to support WebSocket application bindings on different ports

### PR DESCRIPTION
**Pull request by spericas**
New subsection describing enhancement to support WebSocket application bindings on different ports.

Signed-off-by: Santiago Pericasgeertsen <santiago.pericasgeertsen@oracle.com>

### CODE REVIEW

File: docs/mp/websocket.adoc [hunk 1]
```diff
@@ -174,6 +174,47 @@ All endpoint methods in Helidon MP are executed in a separate thread pool, indep
 
 For more information see the link:{helidon-github-tree-url}/examples/webserver/websocket[example].
 
+=== WebSocket Endpoints on Different Ports
+
+The Helidon Reactive Webserver can listen on multiple ports or sockets. This can be useful when APIs for different type of users need to be exposed (such as admin vs. non-admin users). Just like for REST resources, it is possible to expose  WebSocket applications on different ports _provided that the routing paths are different_ --this is due to a constraint in Tyrus, given that it is simply unaware that the endpoints are bound to different ports in the Helidon Webserver.
```
Comment (ljamen) [inline docs/mp/websocket.adoc:179]: In the docs we are referring to it as Reactive Web**S**erver or Helidon Web**S**erver. When referring to a generic web server it's two words, lowercase web server.

https://helidon.io/docs/v3/#/se/webserver

**Review by ljamen — COMMENTED** [review]
Small comment, LGTM.

File: docs/mp/websocket.adoc (hunk 1 above)
Comment (spericas) [reply docs/mp/websocket.adoc:179]: OK, I will change that, but the section title on the left panel reads "Reactive Webserver" as written above.

File: docs/mp/websocket.adoc [hunk 2]
```diff
@@ -174,6 +174,47 @@ All endpoint methods in Helidon MP are executed in a separate thread pool, indep
...
+
+NOTE: In practice, this implies that the value of `@RoutingPath`, or the equivalent entry in config, must be different across sockets to satisfy the restriction in Tyrus.
```
Comment (tjquinno) [inline docs/mp/websocket.adoc:181]: This is not a doc issue, but is there/could there be runtime checking in the code to _make sure_ that the routing paths are different and throw an exception if not?

**Review by tjquinno — COMMENTED** [review]
A question...

File: docs/mp/websocket.adoc (hunk 2 above)
Comment (spericas) [reply docs/mp/websocket.adoc:181]: There is an exception being thrown yes, and the message should be clear.

Comment (tjquinno) [reply docs/mp/websocket.adoc:181]: Nice. Then maybe the doc could mention that the code does this checking and cite the exception users can expect?

Comment (spericas) [reply docs/mp/websocket.adoc]: Added a sentence about the exception.

**Review by tjquinno — APPROVED** [review]

**Pull request merged**